File: sample/src/main/java/com/ebnbin/ebapplication/sample/SampleApplication.java
package com.ebnbin.ebapplication.sample;

import com.ebnbin.eb.util.EBUtil;
import com.ebnbin.ebapplication.context.EBApplication;

public final class SampleApplication extends EBApplication {
    @Override
    public void onCreate() {
        super.onCreate();

        initEBUtil();
    }

    private void initEBUtil() {
        EBUtil.INSTANCE.init(BuildConfig.APPLICATION_ID);
    }
}

File: sample/src/main/java/com/ebnbin/ebapplication/sample/SampleApplication.kt
package com.ebnbin.ebapplication.sample

import com.ebnbin.eb.util.EBUtil
import com.ebnbin.ebapplication.context.EBApplication

class SampleApplication : EBApplication() {
    override fun onCreate() {
        super.onCreate()

        initEBUtil()
    }

    private fun initEBUtil() {
        EBUtil.init(BuildConfig.APPLICATION_ID)
    }
}
